index.html

```diff
@@ -5,22 +5,25 @@
   <head>
     <title>Grimorio interactivo: evocación</title>
     <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;700&family=Grenze+Gotisch:wght@600&display=swap" rel="stylesheet">
   </head>
   <body>
     <nav id="navbar">
       <header>Hechizos de Evocación</header>
       <ul>
-        <li><a class="nav-link" href="#nombre-del-hechizo-1">Nombre del hechizo 1</a></li>
-        <li><a class="nav-link" href="#nombre-del-hechizo-2">Nombre del hechizo 2</a></li>
-        <li><a class="nav-link" href="#nombre-del-hechizo-3">Nombre del hechizo 3</a></li>
+        <li><a class="nav-link" href="#Nombre_del_hechizo_1">Nombre del hechizo 1</a></li>
+        <li><a class="nav-link" href="#Nombre_del_hechizo_2">Nombre del hechizo 2</a></li>
+        <li><a class="nav-link" href="#Nombre_del_hechizo_3">Nombre del hechizo 3</a></li>
+        <li><a class="nav-link" href="#Nombre_del_hechizo_4">Nombre del hechizo 4</a></li>
+        <li><a class="nav-link" href="#Nombre_del_hechizo_5">Nombre del hechizo 5</a></li>
       </ul>
       <footer>
         <p>Copyright 1105 Merlín el Encantador. Todos los derechos reservados.</p>
       </footer>
     </nav>
     
     <main id="main-doc">
-      <section class="main-section" id="nombre-del-hechizo-1">
+      <section class="main-section" id="Nombre_del_hechizo_1">
         <header>Nombre del hechizo 1</header>
         <article>
           <p>Lorem ipsum dolor sit amet consectetur adispicing elit. Lorem ipsum dolor sit amet <strong>consectetur adispicing elit</strong>. Lorem ipsum dolor sit amet consectetur elit. Lorem ipsum dolor sit amet adispicing elit.</p>
```

Comment:
Are you aware that there are [lorem ipsum generators](lipsum.com)? You don't have to reuse the same sentence, you can generate a particular number of words or paragraphs to use as placeholders.

Reply:
I'm aware, I'm aware. I've been using lipsum since forever on print design, and there's even themed generators. I just know the "Lorem ipsum dolor sit amet consectetur adispicing elit" by heart and I type it in whenever I need just a bit of dummy text.